code/datums/weather/weather_types.dm

```diff
@@ -151,7 +153,10 @@
 					else
 						randmutg(H)
 					H.domutcheck()
-	L.rad_act(20,1)
+	if(ismonkey(L) && !L.ckey)
```

Comment:
turn off "hear all emotes" obviously

Or make emotes from mobs without ckeys not show up to every ghost